### PR TITLE
Updated CHANELOG.md for 1.11.0

### PR DESCRIPTION
**Pull request by chethanreddyvelverti**
Updated the ChangeLOG.md file for december release

Please have a look at our [guidelines for contributions](https://github.com/Microsoft/AppCenter-SDK-Android/blob/develop/CONTRIBUTING.md) and consider the following before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.

## Description
Pre-releasing the SDK. Submiting a PR to merge branch 1.10.1 into develop

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # App Center SDK for Android Change Log
 
-## Version 1.10.1 (Under active development)
+## Version 1.10.1
```
Comment (MatkovIvan) [inline CHANGELOG.md]: This update has a feature, so the new version should be `1.11.0`

Comment (chethanreddyvelverti) [reply CHANGELOG.md]: Sure. we will change the version number.

**Comment by jwallra** [comment]
@guperrot Did we actually make this change: https://github.com/Microsoft/AppCenter-SDK-Android/pull/906/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR8 ? 

 I didn't think that we could avoid this because we can't know the size of logs on disk without a prohibitive amount of overhead.

**Comment by guperrot** [comment]
@jwallra it's a best effort simple check.

**Pull request merged**